client/systems.go

```diff
@@ -70,3 +73,24 @@ func (client *Client) ListSystems() ([]System, error) {
 	}
 	return rsp.Systems, nil
 }
+
+// RequestSystemAction issues a request to run the given seed system in the
+// provided mode.
+func (client *Client) RequestSystemAction(systemLabel, actionMode string) error {
+	req := struct {
```

Comment:
we want the option to add other possiblities to the post, also normally all the posts in our APIs tend to have a action field, so maybe somethind like:

```
   Action string  `json:"action"`
   *SystemAction
```

we would set Action="do"